pkg/scan/scan_raw.go

```
@@ -30,6 +30,12 @@ var (
 	handlers *Handlers
 )
 
+// TODO: use this for reade byte array from ICMP req's
+const (
+	icmpHeaderLen = 4
```

golangci-lint (2.12.2)

[error] const icmpHeaderLen is unused

(unused)